samples/nrf9160/modem_shell/src/link/link_shell_pdn.h

```diff
@@ -10,13 +10,24 @@
 #include <shell/shell.h>
 #include <modem/pdn.h>
 
+struct link_shell_pdn_auth {
+	enum pdn_auth method;
+	char *user;
+	char *password;
+};
+
 void link_shell_pdn_init(const struct shell *shell);
 
 int link_shell_pdn_connect(const struct shell *shell, const char *apn_name,
-			    const char *family_str);
+			   const char *family_str,
```

Comment:
Indentation should match with the previous line based on what's seen in nrf code.

Reply:
It does in this particular case, it's github diff preview that messes with tab indentation.

Reply:
ok